mmv1/products/compute/NodeTemplate.yaml

```diff
@@ -145,6 +150,25 @@ properties:
         values:
           - :RESTART_NODE_ON_ANY_SERVER
           - :RESTART_NODE_ON_MINIMAL_SERVERS
+  - !ruby/object:Api::Type::Array
+    name: 'accelerators'
+    description: |
+      List of the type and count of accelerator cards attached to the
+      node template
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::Integer
+          name: 'acceleratorCount'
+          description: |
+            The number of the guest accelerator cards exposed to this
+            node template.
+        # TODO(alexstephen): Change to ResourceRef once AcceleratorType is
+        # created.
```

Comment:
```suggestion
```

instead of having a todo in the files, better to open an issue 
that said, this isnt super necessary regardless as resource ref doesnt functionally do anything